package.json

```diff
@@ -23,9 +23,9 @@
     "url": "https://github.com/project-chip/matter.js.git"
   },
   "scripts": {
-    "clean": "rm -rf dist",
+    "clean": "rm -rf dist; rm -rf out",
     "build": "npm run clean; tsc --project tsconfig.dist-es.json; tsc --project tsconfig.dist-cjs.json; tsc --project tsconfig.dist-dts.json",
-    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
+    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js --coverage",
```

Comment:
to collect the coverage on all files (even those with no unit tests), the parameter is:
--collectCoverageFrom=src/**/*.ts

However it would be more readable to keep all jest configuration in the jest.config.ts file.
The same configuration for code coverage would be:
  collectCoverage: true,
  collectCoverageFrom: ['src/**/*.ts'],

Reference article: https://joshtronic.com/2017/10/24/configuring-jest-to-show-code-coverage-for-all-of-your-files/

Reply:
Ok, I'll update it.  Thx